### PR TITLE
[MPQEditor] Implement createFromDefaultExplorer

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit implements createFromDefaultExplorer command for manual mixed precision task.

Its correctness is tested in https://github.com/Samsung/ONE-vscode/pull/1511.

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1511
Previous draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by seanshpark — DISMISSED** [review]
LGTM

File: src/MPQEditor/MPQEditor.ts [hunk 1]
```diff
@@ -47,6 +47,12 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
         }
       ),
       // Add command registration here
+      vscode.commands.registerCommand(
+        "one.editor.mpq.showFromDefaultExplorer",
```
Comment (dayo09) [inline src/MPQEditor/MPQEditor.ts:51]: May I ask why the command name is 'show*', as far as I understand, it should be 'createMPQJson'...? (THE SAME COMMENT TO https://github.com/Samsung/ONE-vscode/pull/1529)

Comment (stamalakhov) [reply src/MPQEditor/MPQEditor.ts]: @dayo09 
fixed

**Review by dayo09 — APPROVED** [review]
LGTM, Thanks :-D

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**